### DISCUSSION TITLE
code base

### DISCUSSION THREAD

**Discussion by Saty248** [twbs/bootstrap#32010] [Q&A] [answered]
hey everyone i am new to this open source platform and i want to become a contributor for this bootstrap ..can u guys explain me the codebase ..

**Comment by Deckluhm** [comment, marked as answer]
Unless you have specific questions we can help you with, the best way to "learn" the codebase of a project is to:
- Read the docs and try things
- Read the actual source code

If you have good knowledge of JS and (S)CSS basics, Bootstrap source code is pretty straightforward to read and understand.

**Comment by Saty248** [comment]
Okay...thank you
